Add basic notes for the repo (#99)
databricks/databricks-ml-examples: CONTRIBUTING.md

# Adding examples for a new model

> 1. Create a new foler with the model name under `llm-models`
> 2. Add your and your team GitHub handles into the [`CODEOWNERS`](./CODEOWNERS) file. See [documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location). Please keep the list of folder sorted alphabetically.
> 3. Create a `README.md` file in the root of your folder and add frontmatter metadata on top of it.
> 4. Create a branch and push it to this repository either with a fork (if you're not added directly) and create a pull request.

Reviewer: ```suggestion
1. Create a new folder with the model name under `llm-models`.
2. Add your and your team's GitHub handles to the [`CODEOWNERS`](./CODEOWNERS) file. See the [documentation](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-file-location). Please keep the list of folders sorted alphabetically.
3. Create a `README.md` file in the root of your folder and add frontmatter metadata at the top.
4. Create a branch and push it to this repository, either with a fork (if you're not added directly), and create a pull request.
```
